zipline/lib/adjustment.pyx

```diff
@@ -1,12 +1,116 @@
+# cython: embedsignature=True
 from cpython cimport Py_EQ
 
-from pandas import isnull
-from numpy cimport float64_t, uint8_t
+from pandas import isnull, Timestamp
+from numpy cimport float64_t, uint8_t, int64_t
+from numpy import datetime64, float64
 # Purely for readability. There aren't C-level declarations for these types.
 ctypedef object Int64Index_t
 ctypedef object DatetimeIndex_t
 ctypedef object Timestamp_t
 
+# Adjustment kinds.
+MULTIPLY = 0
```

Comment:
Could we define these with `cpdef enum AdjustmentKind:` for clarity. We can then change the usages of `int adjustment_kind` to `AdjustmentKind adjustment_kind`. This compiles the same but ls more clear in the source.